device-catalog/src/data/mod.rs

```diff
@@ -0,0 +1,6 @@
+pub mod android;
```

Comment:
Consider using `include_bytes!` for certificates, and documenting their sources here (as `rustdoc`).

That way all certificates are normal files, rather than embedded in code, and it's very clear it comes from a third-party source.